docker-compose.docker-device.yml

```diff
@@ -0,0 +1,8 @@
+version: '2.1'
```

Comment:
for consistencies sake I think renaming this file would be good, but might not be strictly necessary

Reply:
You are right, thanks!

This file is actually not in use for now - but I plan to use it in integ tests soon :)